cmd/hyperkube/kube-apiserver.go

```diff
@@ -27,8 +27,10 @@ func NewKubeAPIServer() *Server {
 	s := options.NewServerRunOptions()
 
 	hks := Server{
-		SimpleUsage: "apiserver",
-		Long:        "The main API entrypoint and interface to the storage system.  The API server is also the focal point for all authorization decisions.",
+		name:            "apiserver",
```

Comment:
not critical, but if we're going to add `name`s for everything, maybe also add them for the federation- servers and then just simplify `Name()`?